<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContestStatistics</class>
 <widget class="QWidget" name="ContestStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contest Statistics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,3,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Contest</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="contestComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Date from</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="fromDateEdit">
         <property name="displayFormat">
          <string>yyyy-MM-dd</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Date to</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="toDateEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="displayFormat">
          <string>yyyy-MM-dd</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="specificLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Specific</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="specificLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>100</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="multisLabel">
       <property name="text">
        <string notr="true">Multis</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPlainTextEdit" name="multisPlainTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::WidgetWidth</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="multis2Label">
       <property name="text">
        <string notr="true">Multis 2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="multis2PlainTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Infos:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="infosLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Warnings:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="warningsLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Errors:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="errorsLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>contestComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>ContestStatistics</receiver>
   <slot>contestChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromDateEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>ContestStatistics</receiver>
   <slot>fromDateChanged(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toDateEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>ContestStatistics</receiver>
   <slot>toDateChanged(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>specificLineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>ContestStatistics</receiver>
   <slot>specificChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>contestChanged(QString)</slot>
  <slot>fromDateChanged(QDate)</slot>
  <slot>toDateChanged(QDate)</slot>
  <slot>specificChanged()</slot>
 </slots>
</ui>
